vllm/v1/core/scheduler.py

```diff
@@ -134,11 +148,14 @@ def schedule(self) -> "SchedulerOutput":
                 req_index += 1
                 continue
 
-            num_new_tokens = (request.num_tokens_with_spec -
-                              request.num_computed_tokens)
+            num_new_tokens = request.num_tokens - request.num_computed_tokens
```

Comment:
Please revert this line. Although structured and spec decoding together are not supported, I believe each is supported individually.

Reply:
good catch!! I think this came from a bad merge from main.

Reply:
This is reverted now